### PR TITLE
Expand previous editions on history tab

### PR DESCRIPTION
**Pull request by thehenster**
So we can browse the history of a guide we need to be able to open up the previous editions. Previously only the latest edition panel was accessible.

This is part of https://trello.com/c/h4JkfGfP but got missed off.

### CODE REVIEW

File: spec/features/guide_history_spec.rb [hunk 1]
```diff
@@ -59,3 +63,23 @@ def events
...
+
+    expect_edition_to_be_open("Version #2")
+
+    click_link "Version #1"
+
+    expect_edition_to_be_open("Version #1")
+  end
+
+  def expect_edition_to_be_open(title)
+    expect(page).to have_css(".open-edition .panel-heading", text: title)
```
Comment (AndrewVos) [inline spec/features/guide_history_spec.rb]: Is this `open-edition` class only used for tests?

Comment (AndrewVos) [inline spec/features/guide_history_spec.rb]: As it is, this isn't a very good feature test. Rather test that something is appearing on the page?

Comment (AndrewVos) [inline spec/features/guide_history_spec.rb]: Oh I didn't see the title part. I think we just need that assertion

Comment (thehenster) [inline spec/features/guide_history_spec.rb]: Sorry - I don't think I understand about just needing that assertion. Could you rephrase me?

Comment (thehenster) [inline spec/features/guide_history_spec.rb]: Yeah, the CSS class `open-edition` makes it easier to test. Personally I'm comfortable with it. It's a pretty semantic block name and isn't likely to change much and so will be less likely to break the test than, for example, we check which type of chevron is used to determine open-ness. We make little compromises all over to increase the testability of code.

I'm open to suggestions - what would you prefer?

Comment (thehenster) [inline spec/features/guide_history_spec.rb]: rephrase *for me? .. Soz. I miss out words sometimes.

Comment (AndrewVos) [inline spec/features/guide_history_spec.rb]: I meant that just testing for the title is enough in this test. Just realised that the title is displayed for all versions though so that's not really enough.

In a feature test you really want to be testing what actually happens to the page. Testing that something has a class isn't really enough. The test should be asserting that the content of that expanded section is actually visible.
If we deleted all the code inside that panel, in the view, this test would be green.

**Comment by AndrewVos** [comment]
Other than one comment, this lgtm.

**Comment by AndrewVos** [comment]
Didn't you want to rename this view/url as part of this pull request?

File: spec/features/guide_history_spec.rb (hunk 1 above)
Comment (thehenster) [inline spec/features/guide_history_spec.rb]: The selector is checking the title but it's scoped to be nested under `.open-edition`.. so I suppose my assertion is saying.. "If this title heading can be found in an `open-edition` block then consider it to be open". But they could all be open right? My test doesn't say that the previous one is closed. I'll add that - thanks.

Do you feel strongly about not using the `open-edition` to indicate open-ness? Like I was saying above, I find it to be a good compromise.. Perhaps an alternative would be to check if the bootstrap class `.panel-body` is present? It looks like collapsed ones don't have one. Makes sense to me.. A collapsed thing doesn't have a body. Another one could be that it contains some nested `.event` divs? A collapsed thing shouldn't contain any events. What do you think?

**Comment by thehenster** [comment]
:wave: @AndrewVos. I've added a commit for the rework of that test.

**Comment by AndrewVos** [comment]
Nice! Lgtm
